csrc/scheduler/utils.h

```diff
@@ -850,6 +850,10 @@ TensorView* scheduleInputToSkipIntermediates(TensorView* tv);
 
 // Returns true if any of the domains of the tensor is symbolic
 bool isSymbolicTensor(const TensorView* tv);
+
+// For shared memory tensor, replay loop domain transformations to allocation
+// domain
+void replayLoopToAllocation(Fusion* fusion);
```

Comment:
The name and the comment of the function seem vague. Nothing is mentioned that it doesn't silently ignore any tensor that already has an allocation domain. Ideally, it should be clear from the function name only.